fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -331,6 +331,7 @@ public String toString() {
     private long checkerCreationTime = 0;
     private Span txnSpan = null;
     private String traceParent = null;
+    @SerializedName("tci")
     private Set<TabletCommitInfo> tabletCommitInfos = null;
```

Comment:
If this part persists, the transaction commit log will become much bigger.
Have you computed how large it will be if this part persists? Is it acceptable for our current single FE architecture?

Reply:
`Memory cost = 16byte * tablet count in this transaction * max running transaction count`
E.g. if we have 10000 tablets and 10000 running transaction
16byte * 10000 * 10000 = 1.6GB

I can clear this info after transaction become visible.

Reply:
IMO, the main risk is that it will lead to increased latency in writing the bdb log each time the transaction is committed.